### PR TITLE
:sparkles: Introduced return method relationship on returns

### PR DESCRIPTION
**Pull request by zwaans**
# Changes
- introduced return method relationship
- Added relationship to returns
- Made relationship optional so it doesn't destroy the tests

### CODE REVIEW

File: specification/schemas/Return.json
Comment (yoan-myparcel) [inline specification/schemas/Return.json:56]: We need to make it required!

**Review by yoan-myparcel — CHANGES_REQUESTED** [review]

File: specification/schemas/Return.json
Comment (zwaans) [reply specification/schemas/Return.json:56]: @yoan-myparcel then it will break all return tests until the API is implemented.

Comment (zwaans) [reply specification/schemas/Return.json:56]: @yoan-myparcel opened this https://github.com/MyParcelCOM/api-specification/pull/881 so it can be merged after the API gets merged.

**Review by yoan-myparcel — APPROVED** [review]

**Pull request merged**
